docs/producer-configuration.md

```diff
@@ -98,29 +98,29 @@ The **Open** button loads the file.
 
 #### KLG - Key Schema Serializer Config
 
-This configuration element is similar to [KLG - Value Schema Serializer Config](#value-schema-serializer-config), except that it is focused in a key schema instead of a value schema.
+This configuration element is similar to [KLG - Value Schema Serializer Config](#KLG---value-schema-serializer-config), except that it is focused in a key schema instead of a value schema.
 
 Whatever schema you define and configure here will be used as the message key.
 
 #### KLG - Key Schema File Serializer Config
 
-This configuration element is similar to [KLG - Value Schema File Serializer Config](#value-schema-file-serializer-config), except that it is focused in a key schema instead of a value schema.
+This configuration element is similar to [KLG - Value Schema File Serializer Config](#KLG---value-schema-file-serializer-config), except that it is focused in a key schema instead of a value schema.
 
 Whatever schema you define and configure here will be used as the message key.
 
 #### Common table for flattened structure
 
 The four previous configuration elements share the same table, which includes the flattened structure that will be used to generate the random values.
 
-  - **Field Name**: flattened field name composed by all the properties from the root class. Ex: PropClass1.PropClass2.PropClass3.   
+  - **Field Name**: flattened field name composed by all the properties from the root class. Ex: `PropClass1.PropClass2.PropClass3`.   
     **Note**: 
        - The arrays show `[]` at the end. If you want to define a specific size for the array, just type the number inside (i.e. `[10]`). 
        - The maps show `[:]` at the end. If you want to define a specific size for the map, just type the number to the left of the colon (i.e. `[10:]`).        
-  - **Field Type**: field type, such as String, Int, Double, Array, etc.  For a complete list of field types, see [Field types](schemas.md#field-types).
+  - **Field Type**: field type, such as String, Int, Double, Array, etc.  For a complete list of field types, see [Field types](https://github.com/sngular/kloadgen/wiki/schemas#field-types).
   - **Field Length**: field length configuration for the Random Tool. For strings, it indicates the number of characters. For numbers, it indicates the number of digits.
   - **Field Values List**: possible list of values for the field, which will be used by the Random Tool to generate values.
     **Note**: 
-       - For reading context values by default in Jmeter, they must be in the way: ${name_of_variable}
+       - For reading context values by default in Jmeter, they must be in the way: `${name_of_variable}`
```

Comment:
I think than in previous reference, it appears as `JMeter` (`_Sidebar.md`, `architecture.md`, `installation-md` ...) instead of `Jmeter`.